### PR TITLE
Outputs indicator values as proportions

### LINKED ISSUE #208: In PSTR, indicator values should be expressed as proportion
This came up in a conversation with Tilman and Anne. Currently some indicators express percent and others express proportion.

### PR DESCRIPTION
**Pull request by lindadelacombaz**
Closes #208.

Dear @maurolepore,

Here I have changed the code to outputs values as proportions (0-1) instead of percentage (0-100).

- I have : changed the xstr_at_company_level() and do not multiply the value per 100 anymore
- Written a test to test the behavior 
- Changed the snapshot of pstr since the values has changed.

Thanks !! :)

### CODE REVIEW

File: tests/testthat/test-pstr.R [hunk 1]
```diff
@@ -115,3 +115,14 @@ test_that("outputs correct values for edge cases", {
...
 })
+
+test_that("outputs values in proportion", {
+
+  companies <- pstr_companies
+  scenarios <- pstr_scenarios
+
+  out <- pstr(companies, scenarios)
+  expect_true(all(out$value <= 1.0))
+})
```
Comment (maurolepore) [inline tests/testthat/test-pstr.R:126]:
```suggestion
test_that("outputs values in proportion", {
  companies <- pstr_companies
  scenarios <- pstr_scenarios

  out <- pstr(companies, scenarios)
  expect_true(all(out$value <= 1.0))
})
```

**Review by maurolepore — APPROVED** [review]
LGTM.  Thanks!

I made a small change. Please commit it and merge. Note the lil button "commit suggestion"

image


--

BTW, did you forget to change the PR from draft to "ready". It's easy to forget. How can we ensure we don't forget? Maybe we need a checklist or something? 

I wonder what would happen if I assign a reviewer to a draft PR from the `gh` CLI. Maybe it will give me a reminder? To test this I'll open a toy draft PR and assign it to you.
